packages/protocol/src/action-protocol/element-type-hints.ts

```diff
@@ -133,3 +147,119 @@ export namespace SetTypeHintsAction {
         };
     }
 }
+
+/**
+ * Send a Request to the server to check if an element is a valid target
```

Comment:
This doc is confusing imho. This action is not to check whether an element is a valid target, it is to check whether the edge information that we provide already prohibits the creation of a valid model. I'm mainly concerned about the use of `target` in the context of the edge here because I think it adds a lot of confusion. In the tool it is okay because there we also have the target of a click/hover/etc. but here I think we should find something more exact. What do you think?